try:
    result = 45/5

except:
    print('errorheppend')

finally:
    print('finally here')

print('Done')